mnist/training/local/kustomization.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- training.yaml
+
+namespace: kubeflow
+
+generatorOptions:
+  disableNameSuffixHash: true
+
+images:
+- name: training-image
+
+vars:
+- fieldref:
+    fieldPath: data.name
+  name: trainingName
+  objref:
+    apiVersion: v1
+    kind: ConfigMap
+    name: mnist-map-local
+- fieldref:
+    fieldPath: data.pvcName
+  name: pvcName
+  objref:
+    apiVersion: v1
+    kind: ConfigMap
+    name: mnist-map-local
+
+configurations:
+- params.yaml
+
+configMapGenerator:
+- literals:
+  name: mnist-map-local
```

Comment:
syntax error here if use don't run `kustomize edit add configmap mnist-map-local --from-literal=name=mnist-train-local`. maybe you can set a default value here to avoid misunderstanding like below:
```
- name: mnist-map-local
  literals:	
  - name=mnist-train-local
```
please also update others similar as here

Reply:
Thanks Hougang, if set default value, the current kustomize cannot set to new value for configmap, only support to add one metrics.